### DISCUSSION TITLE
No windows ARM64 release?

### DISCUSSION THREAD

**Discussion by NormalHuman-Anything** [timschneeb/GalaxyBudsClient#296] [Releases]
i was wondering why there is no win arm64 release...

**Comment by timschneeb** [comment]
Many users reported that the Windows ARM version did not start properly. Since I wasn't able to reproduce and fix it properly, I stopped distributing them with the other releases.

You can still get up-to-date precompiled ARM binaries for Windows here: https://github.com/ThePBone/GalaxyBudsClient/actions/runs/1431858190 (scroll to the bottom)

**Reply by NormalHuman-Anything** [reply]
Thanks for the help ^^
